packages/debugger/src/context.ts

```diff
@@ -130,3 +131,52 @@ export function createTestContext<Code extends LocaleCode>(config: {
 
   return { scriptConfigs, executor };
 }
+
+export function getDefaultConfig(): {
+  deps: LocaleCode;
+} {
+  return {
+    deps: {
+      ALWAYS_SUCCESS: {
+        dep_type: "code",
+        path: path.join(__dirname, "../bin/always_success"),
```

Comment:
the `context.ts` will be transpiled to `lib/context.js`, we should add `bin` to `package.json`'s `files` to ensure `bin` will be included when published